adafruit_seesaw/seesaw.py

```diff
@@ -255,10 +265,10 @@ def analog_read(self, pin, delay=0.008):
         if pin not in self.pin_mapping.analog_pins:
             raise ValueError("Invalid ADC pin")
 
-        if self.chip_id == _ATTINY8X7_HW_ID_CODE:
-            offset = pin
-        elif self.chip_id == _SAMD09_HW_ID_CODE:
+        self.chip_id == _SAMD09_HW_ID_CODE:
```

Comment:
Simple syntax error here - missing `if`.

I think that's rippling out and causing a lot of the other failures. Might also need some black/pylint-ification of the two files as well.

Reply:
*sigh* sorry i thought it pushed, commited & pushed now